postgresql/hooks/run

```diff
@@ -4,5 +4,4 @@
 export PGDATA={{pkg.svc_data_path}}
 
 exec 2>&1
-exec {{pkg.path}}/bin/postmaster \
-     -c config_file={{pkg.svc_path}}/config/postgresql.conf
+exec {{pkg.path}}/bin/postmaster -c config_file={{pkg.svc_config_path}}/postgresql.conf
```

Comment:
Sure we can probably do that now! My editor just reformatted this line for line length ;)